environment/workspace/modules/networking/vpc-lattice/abtesting/deploymentv2.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: checkout
+spec:
+  template:
+    spec:
+      containers:
+        - name: checkout
+          image: "public.ecr.aws/y1b0a4i8/checkoutv2:latest"
```

Comment:
this image should be built as part of the upstream repo: https://github.com/aws-containers/retail-store-sample-app

Reply:
fixed using new env variable provided by @niallthomson